playbooks/tasks/portal-execute-commands.yml

```diff
@@ -1,7 +1,14 @@
 ---
-# Execute any defined docker commands
-- name: Executing Docker Command
-  include_tasks: tasks/portal-docker-command.yml
+# Execute any defined docker/shell command(s).
+# This loops over all the docker/shell commands listed and executes them one at a
+# time. Variable docker_commands is depreciated, shell_commands should be used
+# instead.
+- name: Executing shell command(s)
+  vars:
+    commands: "{{ (docker_commands | default([])) + (shell_commands | default([])) }}"
+  ansible.builtin.shell: "{{ item }}"
```

Comment:
REQ: Shouldn't this be `tasks/shell-commands-run-subtask.yml`?

Reply:
Currently during deploys we don't log commands' outputs (unless the command fails), so I didn't want to spam deploy logs as the subtask would. Also I didn't want to introduce an unnecessary complexity adding a new "no log" switch to the shell commands task/subtask so I kept here a simple loop/shell task.

Reply:
Currently running docker/shell commands doesn't spam deploy logs with commands outputs unless the command fails.

I can make it reuse the subtask (prioritizing DRY principle) and to keep not spamming logs we would need to add a new flag to not log to ansible logs during deploys, but to log when explicit run commands playbook (or internal playbook) is run which will add another complexity to playbooks.

Or we can keep the playbooks cleaner (prioritizing KISS principle) with this simple loop/shell task.

LMK if you still want to prioritize DRY over KISS?

Reply:
Decided on discord:
- DRY
- keep displaying command output logs